tokio-console/src/view/mod.rs

```diff
@@ -18,11 +19,13 @@ mod tasks;
 pub(crate) use self::styles::{Palette, Styles};
 pub(crate) use self::table::SortBy;
 
-const DUR_LEN: usize = 10;
+// SAFETY: 6 is non-zero.
+const DUR_LEN: NonZeroUsize = unsafe { NonZeroUsize::new_unchecked(6) };
```

Comment:
i don't love that this is `unsafe`. note that `NonZeroUsize::new` is `const fn` in our MSRV, so I believe this could just be 
```suggestion
const DUR_LEN: NonZeroUsize = NonZeroUsize::new(6).expect("6 is non-zero");
```

alternatively, we could just get rid of the `NonZeroUsize`, which i would also be fine about.

Reply:
The unwrap wasn't const in our MSRV I believe.

I changed this back to `Option<usize>`.

Reply:
Yeah, I get the following when trying that code:

```
error: `std::option::Option::<T>::expect` is not yet stable as a const fn
  --> tokio-console/src/view/mod.rs:22:31
   |
22 | const DUR_LEN: NonZeroUsize = NonZeroUsize::new(6).expect("6 is non-zero");
   |        
```

So it seems like while `NonZeroUsize::new()` is stable, we would also need `const Option::unwrap()`, which isn't yet stable.

Another thing I've learnt today.